lib/compiler.js

```diff
@@ -187,13 +186,13 @@ var importExportVisitor = new types.PathVisitor.fromMethodsObject({
         break;
       }
 
-      if (n.SwitchCase.check(stmtPath.value)) {
+      if (stmtPath.value.type === "SwitchCase") {
         assert.strictEqual(bodyPath.name, "consequent");
         insertCharIndex = bodyPath.value[0].start;
         break;
       }
 
-      if (n.Statement.check(stmtPath.value)) {
+      if (! Array.isArray(stmtPath.value)) {
```

Comment:
Just so I understand better can you point to where `Statement.check` is equiv to `Array.isArray`.